### PR TITLE
add rw params

### PR DESCRIPTION
**Pull request by fsx950223**
Use ```model.params``` to keep track of params.
I will add it to saved model and ckpt path.
cc @mingxingtan

### CODE REVIEW

File: efficientdet/main.py [hunk 1]
```diff
@@ -240,10 +241,16 @@ def _can_partition(spatial_dim):
       per_host_input_for_training=tf.estimator.tpu.InputPipelineConfig
       .PER_HOST_V2)
 
+  if FLAGS.model_dir is None:
```
Comment (mingxingtan) [inline efficientdet/main.py:244]: Maybe we can simple mark model_dir as required?

If users get used to specify model_dir=None or not specify it, then they will often have troubles to continue a stopped training job: when they restart the training job, model_dir will change and the job wouldn't start from the previously trained checkpoionts.

WDYT?

File: efficientdet/utils.py [hunk 2]
```diff
@@ -31,6 +32,16 @@
 from tensorflow.python.tpu import tpu_function  # pylint:disable=g-direct-tensorflow-import
 # pylint: disable=logging-format-interpolation
 
+def rw_params(params):
+  os.makedirs(params['model_dir'], exist_ok=True)
+  params_path = os.path.join(params['model_dir'], 'model.params')
+  if os.path.exists(params_path):
+    content = open(params_path,'rb').read()
+    params = json.loads(content.decode(encoding="utf-8"))
```
Comment (mingxingtan) [inline efficientdet/utils.py:40]: Should we use yaml for consistency?

File: efficientdet/main.py (hunk 1 above)
Comment (fsx950223) [reply efficientdet/main.py:244]: I specific model dir by shell in my repo(Also use timestamp).
But for beginners, it's very convenient.
And I believe continue training is an advanced feature, they have ability to specific model dir when they need to restart training jobs.
Two strategies are goods for me.

File: efficientdet/utils.py (hunk 2 above)
Comment (fsx950223) [reply efficientdet/utils.py:40]: Good point.

**Pull request closed without merging**